dotnet/samples/KernelSyntaxExamples/Getting_Started/Step3_Yaml_Prompt.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.IO;
+using System.Reflection;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.PromptTemplate.Handlebars;
+
+/**
+ * This example shows how to create a prompt <see cref="KernelFunction"/> from a YAML resource.
+ */
+// ReSharper disable once InconsistentNaming
+public static class Step3_Yaml_Prompt
+{
+    /// <summary>
+    /// Show how to create a prompt <see cref="KernelFunction"/> from a YAML resource.
+    /// </summary>
+    public static async Task RunAsync()
+    {
+        // Create a kernel with OpenAI chat completion
+        Kernel kernel = Kernel.CreateBuilder()
+            .AddOpenAIChatCompletion(
+                modelId: TestConfiguration.OpenAI.ChatModelId,
+                apiKey: TestConfiguration.OpenAI.ApiKey)
+            .Build();
+
+        // Load prompt from resource
+        var resourceName = "Resources.GenerateStory.yaml";
+        using StreamReader reader1 = new(Assembly.GetExecutingAssembly().GetManifestResourceStream(resourceName)!);
```

Comment:
Can we have the template literal here, without having a Resource file for it?

I could not find this file also "Resources.generateStory.yaml" where is it?

Can we use the EmbeddedResource.cs instead of this complex syntax?

Reply:
The files are in the Resources folder.

I'll switch to template literals

Reply:
Ah, of course I coudn't see them in the solution, we might consider adding them to be visible items in the solution.